src/graphql/schema/buildMutationInputType.ts

```diff
@@ -68,7 +68,7 @@ function buildMutationInputType(name: string, fields: Field[], parentName: strin
       let type: PayloadGraphQLRelationshipType = GraphQLString;
 
       if (Array.isArray(relationTo)) {
-        const fullName = `${combineParentName(parentName, field.label)}RelationshipInput`;
+        const fullName = `${combineParentName(parentName, field.label as string)}RelationshipInput`;
```

Comment:
What happens if this field's label is set to `false`? Would this not create a proper field name, or worse, would it stringify `false` as the name? We should fall back to `field.name` here instead I think.